tuist/tuist
# Resolve warnings for old API being used in unit tests

Resolves https://github.com/tuist/tuist/issues/YYY

### Short description 📝

This PR is about getting rid of the warnings for old API being used in unit tests. 

closes https://github.com/tuist/tuist/issues/510

### Solution 📦 & Implementation 👩‍💻👨‍💻

Replacing the deprecated API with the new ones. 

### How to test it? 

Run unit tests and check the there are no warnings related to unit tests in Xcode's issue navigator 🎊
